core/src/block_sync.rs

```diff
@@ -105,7 +105,7 @@ impl BlockSynchronizer {
 
     /// Create [`Self`] from [`Configuration`]
     pub fn from_configuration(
```

Comment:
```suggestion
    pub fn from_config(
```

we should rename everywhere to use the abbrev